packages/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -323,6 +326,17 @@ public void setMapType(int mapType) {
     googleMap.setMapType(mapType);
   }
 
+  @Override
+  public void setMapStyle(String mapStyle) {
+    if (this.mapStyle.equals(mapStyle)) {
+      return;
+    }
+    this.mapStyle = mapStyle;
+    if (googleMap != null) {
+      googleMap.setMapStyle(new MapStyleOptions(mapStyle));
```

Comment:
Is there any error handling we might need to do with malformed style json blobs?

Reply:
Added a null check